### ISSUE TITLE
Pre-work Checklist: Developer: Steven Tran

### ISSUE THREAD

**Issue by steven-positive-tran** [hackforla/website#3826]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by steven-positive-tran** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

onboarding 3 hours
first meeting 2 hours

docker github setup 2 hours
first issue 2 hourss

availablity normally after 6 or 7 any day. can do earlier if scheduled and told early but weekdays are a no.

**Comment by ExperimentsInHonesty** [comment]
Hi @steven-positive-tran I am reopening this issue until you have had a chance to use the progress report when an issue you are working on takes more than a week
```
Provide Update
1. Progress
2. Blockers
3. Availability
4. ETA
```

**Comment by steven-positive-tran** [comment]
Progress: "What is the current status of your project? What have you completed and what is left to do?"
Completed 
    Blockers: "Difficulties or errors encountered."
 None
    Availability: "How much time will you have this week to work on this issues"
This week after 4pm

**Comment by ExperimentsInHonesty** [comment]
Hi @steven-positive-tran I think you have misunderstood.  We don't need you to do the update on this issue, we are holding this issue open until you take on an issue that takes longer than a week to complete (usually an issue with 2pts) and that then you will be providing an update on that.

**Comment by steven-positive-tran** [comment]
#4050

Progress: Started, Researching
Blockers: "Difficulties or errors encountered.": None yet
Availability: 7pm-12pm Anyday
ETA: One week

**Comment by ExperimentsInHonesty** [comment]
@steven-positive-tran I am reopening this issue until you have been assigned to a development issue long enough to make an in progress update.  It can will stay open until you get an issue that takes longer than a week to complete (likely a medium or large size issue).  That way, we know you can make detailed progress updates.  In the meantime, if you get reminders to update this issue, just let us know what size and number issue you are assigned to at the time.  That way, we have a record (in one place) of your progress

**Comment by ExperimentsInHonesty** [comment]
@steven-positive-tran Friendly reminder to move on to a Large size issue and not to do any more medium size issues after the one you are on now #4050.

**Comment by jdingeman** [comment]
Hi @steven-positive-tran, I wanted to check in. It looks like you still need a large issue. You can find those here: https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22

**Comment by roslynwythe** [comment]
Hi @steven-positive-tran I know that you have been working on other issues and we appreciate your contributions.  We are in the process of transforming the prework issue into a tracking issue that will remain open indefinitely, and we ask that you leave an update on your progress at least every two weeks, using the outline presented in the above comment.  Thank you.

**Comment by steven-positive-tran** [comment]
#3223
    Progress: Waiting for DR
    Blockers: None
    Availability: Avail all day from 12pm to 6
    ETA: End of this week if no update I'll move on to another issue.

**Comment by HackforLABot** [comment]
## Activity Log: steven-positive-tran
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- steven-positive-tran opened: https://github.com/hackforla/website/issues/3826 at 01/16/2023, 07:47 PM PST
- steven-positive-tran assigned: https://github.com/hackforla/website/issues/3826 at 01/16/2023, 08:04 PM PST
- steven-positive-tran assigned: https://github.com/hackforla/website/issues/3823 at 01/22/2023, 11:33 PM PST
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/3823#issuecomment-1399924466 at 01/22/2023, 11:45 PM PST
- steven-positive-tran opened: https://github.com/hackforla/website/pull/3864 at 01/23/2023, 08:19 PM PST
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/3826#issuecomment-1401380135 at 01/23/2023, 08:25 PM PST
- steven-positive-tran closed as completed: https://github.com/hackforla/website/issues/3826#event-8340495657 at 01/23/2023, 08:25 PM PST
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/3864#issuecomment-1403040787 at 01/24/2023, 07:13 PM PST
- steven-positive-tran PR was merged: https://github.com/hackforla/website/pull/3864#event-8366877325 at 01/26/2023, 10:47 AM PST
- steven-positive-tran issue closed by PR 3864: https://github.com/hackforla/website/issues/3823#event-8366877522 at 01/26/2023, 10:47 AM PST
- steven-positive-tran assigned: https://github.com/hackforla/website/issues/2148 at 01/28/2023, 01:21 AM PST
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/2148#issuecomment-1407352242 at 01/28/2023, 01:23 AM PST
- steven-positive-tran opened: https://github.com/hackforla/website/pull/3896 at 01/29/2023, 09:18 PM PST
- steven-positive-tran PR was closed: https://github.com/hackforla/website/pull/3896#event-8405651053 at 01/31/2023, 06:34 PM PST
- steven-positive-tran unassigned: https://github.com/hackforla/website/issues/2148#issuecomment-1411369489 at 01/31/2023, 06:35 PM PST
- steven-positive-tran assigned: https://github.com/hackforla/website/issues/3731#issuecomment-1338233914 at 01/31/2023, 11:26 PM PST
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/3731#issuecomment-1411583859 at 01/31/2023, 11:29 PM PST
- steven-positive-tran opened: https://github.com/hackforla/website/pull/3947 at 02/07/2023, 12:36 AM PST
- steven-positive-tran PR was merged: https://github.com/hackforla/website/pull/3947#event-8474176862 at 02/08/2023, 04:10 PM PST
- steven-positive-tran issue closed by PR 3947: https://github.com/hackforla/website/issues/3731#event-8474177017 at 02/08/2023, 04:10 PM PST
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/3826#issuecomment-1445061763 at 02/25/2023, 03:37 AM PST
- steven-positive-tran closed as completed: https://github.com/hackforla/website/issues/3826#event-8607783050 at 02/25/2023, 03:37 AM PST
- steven-positive-tran issue closed by PR 3947: https://github.com/hackforla/website/issues/3731#event-8608842827 at 02/25/2023, 07:06 PM PST
- steven-positive-tran assigned: https://github.com/hackforla/website/issues/2216#issuecomment-1301633567 at 02/26/2023, 03:40 AM PST
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/2216#issuecomment-1445338918 at 02/26/2023, 03:40 AM PST
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4048#issuecomment-1445791820 at 02/26/2023, 10:44 PM PST
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4048#pullrequestreview-1315009675 at 02/26/2023, 11:15 PM PST
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/2216#issuecomment-1449268122 at 02/28/2023, 07:10 PM PST
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/2216#issuecomment-1457713055 at 03/06/2023, 11:57 PM PST
- steven-positive-tran opened: https://github.com/hackforla/website/issues/4146 at 03/10/2023, 12:53 AM PST
- steven-positive-tran closed as completed: https://github.com/hackforla/website/issues/2216#event-8716052303 at 03/10/2023, 12:56 AM PST
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/2216#issuecomment-1463479166 at 03/10/2023, 12:57 AM PST
- steven-positive-tran reopened: https://github.com/hackforla/website/issues/2216#issuecomment-1463479166 at 03/10/2023, 12:57 AM PST
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4150#issuecomment-1465305198 at 03/12/2023, 02:32 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4150#pullrequestreview-1336203403 at 03/12/2023, 08:31 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4135#issuecomment-1465467712 at 03/12/2023, 08:49 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4135#pullrequestreview-1336366713 at 03/13/2023, 12:20 AM PDT
- steven-positive-tran closed as completed: https://github.com/hackforla/website/issues/2216#event-8763524582 at 03/15/2023, 11:50 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4222#issuecomment-1477363764 at 03/21/2023, 12:04 AM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/4050#issuecomment-1477375270 at 03/21/2023, 12:18 AM PDT
- steven-positive-tran assigned: https://github.com/hackforla/website/issues/4050#issuecomment-1477375270 at 03/21/2023, 12:19 AM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/3826#issuecomment-1477379815 at 03/21/2023, 12:24 AM PDT
- steven-positive-tran closed as completed: https://github.com/hackforla/website/issues/3826#event-8802140708 at 03/21/2023, 12:24 AM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4222#pullrequestreview-1351941564 at 03/22/2023, 01:23 AM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/4050#issuecomment-1483773952 at 03/25/2023, 02:20 AM PDT
- steven-positive-tran unassigned: https://github.com/hackforla/website/issues/4050#issuecomment-1483943633 at 04/02/2023, 05:43 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/4050#issuecomment-1493490600 at 04/02/2023, 05:45 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/4146#issuecomment-1499985059 at 04/06/2023, 11:30 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4453#issuecomment-1501391253 at 04/09/2023, 09:58 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4453#pullrequestreview-1377106427 at 04/09/2023, 10:41 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4444#issuecomment-1501433358 at 04/09/2023, 11:03 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4444#pullrequestreview-1378492428 at 04/10/2023, 10:52 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4465#issuecomment-1502725921 at 04/10/2023, 10:55 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4465#issuecomment-1502726922 at 04/10/2023, 10:56 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4465#pullrequestreview-1378546556 at 04/10/2023, 11:44 PM PDT
- steven-positive-tran assigned: https://github.com/hackforla/website/issues/3223#issuecomment-1453087219 at 04/10/2023, 11:54 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/3223#issuecomment-1502780083 at 04/10/2023, 11:59 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4453#pullrequestreview-1380344792 at 04/11/2023, 07:36 PM PDT
- steven-positive-tran opened: https://github.com/hackforla/website/issues/4485 at 04/11/2023, 08:08 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4524#issuecomment-1510661665 at 04/16/2023, 09:01 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4539#issuecomment-1512410552 at 04/17/2023, 09:16 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4539#pullrequestreview-1389285419 at 04/17/2023, 09:55 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4524#pullrequestreview-1389291971 at 04/17/2023, 10:04 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/3223#issuecomment-1512529366 at 04/17/2023, 11:46 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/3223#issuecomment-1527132538 at 04/28/2023, 12:43 AM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4586#issuecomment-1530858762 at 05/01/2023, 09:27 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4586#pullrequestreview-1408405452 at 05/01/2023, 10:24 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4603#issuecomment-1537730865 at 05/07/2023, 09:31 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4603#pullrequestreview-1418012284 at 05/09/2023, 12:06 AM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/3223#issuecomment-1541059833 at 05/09/2023, 05:33 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4644#issuecomment-1541126381 at 05/09/2023, 06:16 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4643#issuecomment-1541126783 at 05/09/2023, 06:16 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/3223#issuecomment-1541303322 at 05/09/2023, 08:41 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4644#pullrequestreview-1419845948 at 05/09/2023, 11:13 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4643#pullrequestreview-1419860877 at 05/09/2023, 11:27 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/3223#issuecomment-1543385011 at 05/10/2023, 10:59 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4644#pullrequestreview-1425300042 at 05/12/2023, 06:25 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4646#pullrequestreview-1425311497 at 05/12/2023, 07:48 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4667#issuecomment-1547027156 at 05/14/2023, 04:29 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4661#issuecomment-1547027828 at 05/14/2023, 04:31 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4661#pullrequestreview-1427756154 at 05/15/2023, 10:42 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4667#pullrequestreview-1427761851 at 05/15/2023, 10:46 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4661#issuecomment-1549076602 at 05/15/2023, 11:42 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/3223#issuecomment-1571334718 at 05/31/2023, 09:51 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/3223#issuecomment-1577979321 at 06/05/2023, 11:16 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4789#issuecomment-1579917106 at 06/06/2023, 10:22 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4772#issuecomment-1579918469 at 06/06/2023, 10:23 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4789#pullrequestreview-1473912919 at 06/11/2023, 06:38 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4789#pullrequestreview-1473913180 at 06/11/2023, 06:38 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4817#issuecomment-1590221979 at 06/13/2023, 04:56 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4796#issuecomment-1590222578 at 06/13/2023, 04:57 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4796#pullrequestreview-1480035542 at 06/14/2023, 11:07 AM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4817#pullrequestreview-1480276467 at 06/14/2023, 01:47 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4866#issuecomment-1599763198 at 06/20/2023, 05:15 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4866#pullrequestreview-1492066112 at 06/21/2023, 04:18 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4882#issuecomment-1608159547 at 06/26/2023, 01:04 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4878#issuecomment-1608185152 at 06/26/2023, 01:17 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4878#pullrequestreview-1499538076 at 06/26/2023, 02:51 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4882#pullrequestreview-1499584025 at 06/26/2023, 03:26 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4882#pullrequestreview-1499584271 at 06/26/2023, 03:27 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/4485#issuecomment-1634992958 at 07/13/2023, 03:23 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/4945#issuecomment-1638647801 at 07/17/2023, 11:21 AM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/4945#pullrequestreview-1533669066 at 07/17/2023, 01:42 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/issues/3826#issuecomment-1648746440 at 07/24/2023, 04:14 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/5095#issuecomment-1652771792 at 07/26/2023, 06:16 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/5096#issuecomment-1652773708 at 07/26/2023, 06:19 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/5095#pullrequestreview-1548858233 at 07/26/2023, 06:43 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/5096#pullrequestreview-1548866475 at 07/26/2023, 06:54 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/5095#pullrequestreview-1553000604 at 07/28/2023, 02:55 PM PDT
- steven-positive-tran closed as completed: https://github.com/hackforla/website/issues/3302#event-10004012337 at 08/03/2023, 04:51 PM PDT
- steven-positive-tran reopened: https://github.com/hackforla/website/issues/3302#event-10004012337 at 08/03/2023, 04:51 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/5133#issuecomment-1664800470 at 08/03/2023, 05:34 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/5125#issuecomment-1664806425 at 08/03/2023, 05:37 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/5133#pullrequestreview-1562031900 at 08/03/2023, 05:47 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/5125#pullrequestreview-1562034676 at 08/03/2023, 05:52 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/5142#issuecomment-1666285406 at 08/04/2023, 04:48 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/5142#pullrequestreview-1563671979 at 08/04/2023, 04:56 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/5152#issuecomment-1668724488 at 08/07/2023, 05:03 PM PDT
- steven-positive-tran commented on: https://github.com/hackforla/website/pull/5155#issuecomment-1668725243 at 08/07/2023, 05:04 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/5152#pullrequestreview-1566293761 at 08/07/2023, 05:09 PM PDT
- steven-positive-tran submitted review: https://github.com/hackforla/website/pull/5155#pullrequestreview-1566298238 at 08/07/2023, 05:17 PM PDT
